utils/src/test/scala/cromwell/pipeline/utils/TestContainersUtils.scala

```diff
@@ -27,16 +25,16 @@ object TestContainersUtils {
   private lazy val mongoConfig = ConfigFactory.load().getConfig("database.mongo")
   private def mongoPort: Int = mongoConfig.getInt("port")
 
-  class MongoContainer(dockerImageName: String) extends GenericContainer[MongoContainer](dockerImageName)
-
-  def getMongoContainer(mongoImageName: String = "mongo"): MongoContainer =
```

Comment:
Why did you remove this method? It's nice to have one method that gives mongo container as we did with postgres

Reply:
I thought it's redundant, but I've changed my code and it's useful now